### DISCUSSION TITLE
colors.setBrand is not a function

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#9850] [CLI - Electron mode] [answered]
It appears in the latest version of Quasar 2.0 there is no method 'colors.setBrand' which was previously there for years. What happened to the dynamic method? Is it expected to com back?

```
import { colors } from 'quasar';
colors.setBrand('primary', state.primaryColor);
```

**Comment by rstoenescu** [comment, marked as answer]
Part of the upgrade guide. I highly recommend reading top to bottom.
https://quasar.dev/start/upgrade-guide#color-utils

**Reply by JokerMartini** [reply]
much appreciated. thank you
